### PR TITLE
Task/cast date attr types datetime in sink

### PR DESCRIPTION
**Pull request by AlvaroVega**
fix for https://github.com/telefonicaid/fiware-cygnus/issues/2020

example
entity

column
column

row
row

### CODE REVIEW

**Review by mrutid — APPROVED** [review]
LGTM

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -0,0 +1 @@
+- [cygnus-ngsi] [MongoSink] Cast to mongo Date type all NGSI DateTime attributes in colum mode (#2020)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- [cygnus-ngsi][MongoSink] Cast to mongo Date type all NGSI DateTime attributes in column mode (#2020)
```

**Review by fgalan — APPROVED** [review]
LGTM

File: cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMongoSink.java [hunk 2]
```diff
@@ -217,6 +217,27 @@ private void persistAggregation(NGSIGenericAggregator aggregator) throws CygnusP
         ArrayList<Document> aggregation = new ArrayList<>();
         for (int i = 0 ; i < jsonObjects.size() ; i++) {
             BasicDBObject basicDBObject = BasicDBObject.parse(jsonObjects.get(i).toString());
+            for (String key : basicDBObject.keySet()) {
+                try {
+                    ArrayList<JsonElement> values;
+                    if (rowAttrPersistence) {
+                        values = aggregator.getAggregation().get(NGSIConstants.ATTR_TYPE);
```
Comment (mrutid) [inline cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMongoSink.java:224]: This is seems to be constant on every iteration

**Review by mrutid — COMMENTED** [review]
Something weird in row mode.

**Pull request merged**